Fails to build with Node v4.1.0

The problem seems related to the `msgpack` module. I've opened an issue there too:
https://github.com/msgpack/msgpack-node/issues/27

```
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/nvm/versions/node/v4.1.0/bin/node" "/usr/local/Cellar/nvm/versions/node/v4.1.0/bin/npm" "install" "zerorpc"
npm ERR! node v4.1.0
npm ERR! npm  v2.14.3
npm ERR! code ELIFECYCLE

npm ERR! msgpack@0.2.6 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the msgpack@0.2.6 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the msgpack package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls msgpack
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Ale/Dropbox/Docs/Good Karma/Projects/Distilled/code/back/koa/npm-debug.log
```

What worries me a bit, looking at the other `msgpack` issues is that the build seems to fail on many Node versions: https://github.com/msgpack/msgpack-node/issues/

Should this module be swapped for another Node JS version of msgpak, or be substituted entirely with `JSON.stringify` and `JSON.parse`? The latter would produce slightly longer strings, but should be way faster with current implementation (as mentioned in [`msgpack-node` docs](https://github.com/msgpack/msgpack-node/blob/master/README.md)).

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
-    - "0.12"
+  - "0.12"
+  - "4"
+env:
+  - CXX="g++-4.8"
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+      - gcc-4.8
 before_install:
   - sudo apt-get update
   - sudo apt-get install libzmq3-dev

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "zerorpc",
-    "version": "0.9.5",
+    "version": "0.9.6",
     "main": "./index.js",
     "author": "François-Xavier Bourlet <bombela+zerorpc@gmail.com>",
     "description": "A port of ZeroRPC to node.js",
@@ -29,7 +29,7 @@
 
     "dependencies": {
         "underscore": "1.3.3",
-        "msgpack": "0.2.6",
+        "msgpack": "1.0.0",
         "node-uuid": "1.3.3",
         "zmq": "2.x"
     },

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
 
     "dependencies": {
         "underscore": "1.3.3",
-        "msgpack": "1.0.0",
+        "msgpack": "1.0.1",
         "node-uuid": "1.3.3",
         "zmq": "2.x"
     },

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "zerorpc",
-    "version": "0.9.6",
+    "version": "0.9.5",
     "main": "./index.js",
     "author": "François-Xavier Bourlet <bombela+zerorpc@gmail.com>",
     "description": "A port of ZeroRPC to node.js",

--- a/package.json
+++ b/package.json
@@ -29,7 +29,7 @@
 
     "dependencies": {
         "underscore": "1.3.3",
-        "msgpack": "1.0.1",
+        "msgpack": "1.0.2",
         "node-uuid": "1.3.3",
         "zmq": "2.x"
     },

